lib/tasks/lint.rake

```diff
@@ -0,0 +1,4 @@
+desc "Lint Ruby"
```

Comment:
Since we're not expecting to define any other tasks, could we just have it inline in the `Rakefile`?